en/docs/guides/tenants/email-account-locking.md

```diff
@@ -0,0 +1,198 @@
+# Configure Email Notifications for Account Locking
+
+Once you have configured WSO2 Identity Server for user account locking,
+you can also configure the WSO2 IS to email to the user's email address
+when the user account is locked. To configure this, follow the steps
+below.
+
+Enable the email sending configurations of the WSO2 Identity Server as explained below.
+
+{! fragments/configure-email-sending.md !}
+
+!!! tip
+    The email template used to send the email notification for
+    account locking is the **AccountLock** template and the template
+    used for account disabling is the **AccountDisable** template. You
+    can edit and customize the email template. For more information on
+    how to do this, see [Customize Automated
+    Emails](../../../guides/tenants/customize-automated-mails).
+
+WSO2 Identity Server uses separate email templates for notifying, 
+
+- Account locking by administrator
+- Account unlocking by administrator 
+
+Add the following email templates by referring to the instructions in [Customize Automated Emails](../../../guides/tenants/customize-automated-mails).
+
+Following are the sample email templates.
```

Comment:
these email templates can already be viewed from the management console. therefore instead of having the email template added in the doc, let's mention the instructions to view those templates